### DISCUSSION TITLE
How do I get the PRO highlight badge?

### DISCUSSION THREAD

**Discussion by HackerstoyanovBG** [community/community#41725] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I earn the **PRO** highlight badge?

**Comment by MitkoVtori** [comment, marked as answer]
by buying [GitHub Pro](https://docs.github.com/en/get-started/learning-about-github/githubs-products#github-pro)

**Comment by codenay** [comment]
You can sign up for the GitHub Student Developer Pack with your student email if you are a student.

**Comment by dev-aniketj** [comment]
I applied for the github student pack and it is approved
I don’t have a pro badge on my github profile
https://github.com/dev-aniketj

**Reply by MitkoVtori** [reply]
I don't think the github student pack gives you badge, but I'm sure it gives you pro. You can test to see if you received the pack.
